robottelo/constants.py

```diff
@@ -283,7 +283,7 @@
 DEFAULT_CV = "Default Organization View"
 DEFAULT_PTABLE = "Kickstart default"
 DEFAULT_SUBSCRIPTION_NAME = (
-    'Red Hat Enterprise Linux Server Entry Level, Self-support')
+    'Red Hat Enterprise Linux Server, Premium (Physical or Virtual Nodes)')
```

Comment:
was working on this last night and was puzzled why it suddenly started to work :D (haven't noticed I already pulled this commit) :+1: